ajax/libs/respond.js/package.json

```diff
@@ -24,5 +24,13 @@
       "url": "https://github.com/scottjehl/Respond"
     }
   ],
-  "namespace": "respond"
+  "namespace": "respond",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/scottjehl/Respond.git",
+    "basePath": "dest",
+    "files": [
+      "respond*.js"
```

Comment:
All the css files are in the test folder and all named 'test*.css', is it ok to add them in autoupdate?